backend/src/main/kotlin/pl/akai/fillist/web/spotifywrapper/playlists/SpotifyPlaylistsService.kt

```diff
@@ -23,4 +23,9 @@ class SpotifyPlaylistsService @Autowired constructor(
             .retrieve()
             .bodyToMono(SpotifyPlaylist::class.java)
     }
+
+    fun getPlaylistDetails(playlistId: String): Mono<SpotifyPlaylist> {
```

Comment:
Duplicated at #54. Wait for the pull request from this issue (https://github.com/akai-org/fillist/pull/66) to be merged with the main, and then integrate it.